src/components/homeMain/HomeMain.js

```diff
@@ -0,0 +1,48 @@
+        <div className="p-3 h-50vh explore-items w-100per img-cover border-radius-2rem"><img
+          className="h-100per w-100per img-cover hover-img"
+          src="https://images.unsplash.com/photo-1563394779868-b02c503851fa?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
+          alt="main" /></div>
+        <div className="p-3 h-50vh explore-items"><img className="h-100per w-100per img-cover hover-img"
+          src="https://images.unsplash.com/photo-1555765093-698c20d82b2d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1211&q=80"
+          alt="main" /></div>
+      </div>
+      <hr />
+      <div className="catagory w-100per flex-center-center flex-wrap p-2">
+        <div className="catagory-items m-1 p-1 relative">
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
+        </div>
+        <div className="catagory-items m-1 p-1 relative">
+
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
+        </div>
+        <div className="catagory-items m-1 p-1 relative">
+
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
+        </div>
+        <div className="catagory-items m-1 p-1 relative">
+
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
+        </div>
+        <div className="catagory-items m-1 p-1 relative">
+
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
+        </div>
+        <div className="catagory-items m-1 p-1 relative">
+
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
```

Comment:
All these are repetitive codes. Use map function. This will reduce your lines of code

Reply:
I will rectify that, thanks for pointing out.